### PR TITLE
[#8] Maven project init

### PR DESCRIPTION
**Pull request by RisteJovanoski**

### CODE REVIEW

File: self-service-registartion/pom.xml [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
+            <artifactId>tomcat-servlet-api</artifactId>
+            <version>7.0.30</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.jersey.bundles</groupId>
```
Comment (emiovska) [inline self-service-registartion/pom.xml]: Why we need glassfish dependency here?

File: self-service-registartion/pom.xml [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
     <version>0.0.1-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <packaging>war</packaging>
     <name>MM-Self-Service-Registration</name>
 
+    <dependencies>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
```
Comment (emiovska) [inline self-service-registartion/pom.xml]: Extract dependencies version in maven properties. It is better for future dependency upgrade.
Apply this to all dependencies version param.

File: self-service-registartion/pom.xml [hunk 3]
```diff
@@ -1,13 +1,54 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
```
Comment (emiovska) [inline self-service-registartion/pom.xml]: Why we need this dependency?

**Review by emiovska — CHANGES_REQUESTED** [review]

File: self-service-registartion/pom.xml (hunk 3 above)
Comment (RisteJovanoski) [reply self-service-registartion/pom.xml]: The dependency will be removed

File: self-service-registartion/pom.xml (hunk 1 above)
Comment (RisteJovanoski) [reply self-service-registartion/pom.xml]: The dependency will be replaced with more specific dependency

File: self-service-registartion/pom.xml (hunk 2 above)
Comment (RisteJovanoski) [reply self-service-registartion/pom.xml]: This dependency will be removed because the application uses REST.

**Review by emiovska — APPROVED** [review]

**Pull request merged**